src/components/Routes.jsx

```diff
@@ -20,6 +20,7 @@ import BecomeSponsorPage from '../pages/BecomeSponsorPage';
 import ContactUsPage from '../pages/ContactUsPage';
 import ZooMeaningPage from '../pages/ZooMeaningPage';
 import Cutco from '../pages/Cutco';
+import SponsortoEventRedirect from '../pages/SponsortoEventRedirect'
```

Comment:
Just for the sake of consistency, for CamelCase it should be `SponsorToEventRedirect`

Reply:
done!